migrations/Version202311241512382260_taoQtiTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoTests\models\runner\plugins\PluginRegistry;
+use oat\taoTests\models\runner\plugins\TestPlugin;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ *
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202311241512382260_taoQtiTest extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Register plugin to prevent session concurrency';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $registry = PluginRegistry::getRegistry();
+        if (!$registry->isRegistered('taoQtiTest/runner/plugins/controls/session/preventConcurrency')) {
```

Comment:
We can store `'taoQtiTest/runner/plugins/controls/session/preventConcurrency'` in a separate class variable to avoid duplications 🙂
But not critical